src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -3945,8 +3986,83 @@ video::bQpwS2nMAZY[youtube, width=700, height=400 start=0, position=center]
 
 *Visdom_Real_time_Digit_Classification:* shows an example of realtime plotting using the Visdom server for training a convolutional neural network (CNN) for MNIST digit classification.
 
+*Check_Visdom_Support:* checks if the user wants (or not) start the Visdom service.
+
 A demo video of these workflows is available in https://www.youtube.com/user/ActiveEonCompany[ActiveEon Youtube Channel^].
 
+==== Tensorboard
+
+It provides  the visualization and tooling needed for machine learning experimentation, such as tracking and visualizing metrics such as loss and accuracy, visualizing the model graph (ops and layers), viewing histograms of weights, biases, or other tensors as they change over time, projecting embeddings to a lower dimensional space, displaying images, text, and audio data and others.
+
+===== Tensorboard_Service_Start
+
+*Task Overview:* Start the TensorBoard server as a service.
+
+*Task Variables:*
+
+.Tensorboard_Service_Start variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `SERVICE_ID`
+| The id of the TensorBoard service.
+| String (default="Tensorboard")
+| `INSTANCE_NAME`
+| The instance name of the server to use to broadcast visualization
```

Comment:
of the server to use to broadcast visualization => of the server to be used to broadcast the visualization